content/blog/2024-01-30-flash-framework.Rmd

```diff
@@ -0,0 +1,67 @@
+---
+title: Alerting Systems are Incompatible with Modern Public Health Data Monitoring 
+author: Ananya Joshi
+date: 2024-01-01
+tags:
+  - flash
+authors:
+  - ajoshi
+heroImage: blog_alerts_hero.png
+heroImageThumb: blog_alerts_thumb.png
+output:
+  blogdown::html_page:
+    toc: true
+---
+Insights from public health data can keep communities safe. However, identifying these insights in large volumes of modern public health data can be laborious <sup>[1](#link1)</sup>. As a result, over the past few decades, public health agencies have built monitoring systems, like [ESSENCE](https://www.cdc.gov/nssp/new-users.html) (CDC), [EIOS](https://www.who.int/initiatives/eios) (WHO), and [DHIS2](https://dhis2.org/) (WHO), where users can set custom statistical alerts and then investigate these alerts using data visualizations <sup>[2](#link2)</sup>. These alerting systems largely follow the following formula <sup>[3](#link3)</sup> as shown in Fig 1.:
+
+
+<center>
+![**Fig 1** Standard Approach for Alerting Systems](/blog/2024-01-30-flash-framework/image3.png)</center>
+
+1. Predict a value that you expect the data to look like 
+2. Calculate the difference between your observed and predicted values 
+3. Contextualize the difference (e.g., to account for the variability in the data)
+4. Send alerts when this value passes a threshold
+
+Using this formula, if we assume that our data follows a bell curve, we can calculate the difference between our prediction (which might be the average of the data) and what we observe and divide by the standard deviation of the data. If this value exceeds a specific threshold of 3, then the probability that we would observe that data point in that bell curve would be less than 1%, which means that it is quite surprising. By setting this threshold high, only a small percentage of points are identified as outliers and result in alerts.
```

Comment:
nit(s) -- take these suggestions with a grain of salt, im not sure if im capturing all of your intent.  i am also not clear on where the "1%" comes from or how it relates to the threshold of 3.
```suggestion
Using this formula, if we assume that our data follows the bell curve of a normal distribution, we can calculate the difference between our prediction (which might be the average of the data) and what we observe, and divide by the standard deviation. If this value exceeds a specific threshold (here we use 3), then the probability that we would observe that data point in that bell curve would be less than 1%, which means that it is quite surprising. By setting this threshold high, only a small percentage of points are identified as outliers and thus result in alerts.
```

Reply:
Ah ! so for the normal distribution specifically, 99% of the data is contained within 3 standard deviations, which means data beyond those 3 deviations (this is a continuous distribution which can take any value between +-inf) is very unlikely (and only <1% of the distribution takes on those values). Let me know if that doesn't make sense!